src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -60,10 +65,30 @@ public Result Result(string query, IPublicAPI api)
                 var noArgumentScoreModifier = 5;
                 score += noArgumentScoreModifier;
             }
+            else
+            {
+                // To Filter PWAs when the user searches for the main application
+                string edgePWA = "msedge_proxy.exe";
+                string chromePWA = "chrome_proxy.exe";
+                bool isAppPWA = FullPath.Contains(edgePWA) || FullPath.Contains(chromePWA);

```

Comment:
It's not a good idea to filter only edge and chrome. Because what is with other chromium based browser like brave?